webapp/src/reducers/index.ts

```diff
@@ -17,7 +17,7 @@ const reducers = combineReducers({
     createTaskModalSlice,
     subscriptionModalSlice,
     projectDetailsSlice,
-    userConnectedSlice,
```

Comment:
shouldn't we remove this reducer as well `userConnectedSlice` if not used anywhere?